qtconsole/console_widget.py

```diff
@@ -2183,9 +2183,9 @@ def _insert_plain_text(self, cursor, text, flush=False):
                     format = self._ansi_processor.get_format()
                     if not (hasattr(cursor,'_insert_mode') and cursor._insert_mode):
                         pos = cursor.position()
-                        remain=self._get_line_end_pos()-pos
+                        remain=self._get_line_end_pos()-pos # could be negative ! As not taking int account last \n ?
```

Comment:
```suggestion
                        remain = self._get_line_end_pos() - pos  # It could be negative! As not taking into account the last \n?
```

Add spaces around operators and improve comment.